node_parse.go

```diff
@@ -347,6 +351,10 @@ func (n *node) addFlagArg(sf reflect.StructField, val reflect.Value) error {
 		i.defstr = defstr
 	} else if val.Kind() == reflect.Slice {
 		i.defstr = ""
+	} else if val.Kind() != reflect.Ptr && val.CanAddr() {
+		if val.Addr().Interface() != reflect.Zero(val.Type()).Interface() {
```

Comment:
What's an example use-case of this? Looks like its when `sf.Type != val.Type`

Reply:
custom types like `RepeatedStringOpt` don't work without this.
eg 
```go
type root struct {
   Volume opts.RepeatedStringOpt
}
func main() {
  r := &root{}
  opts.New(r).Parse()
}
// val := reflect.ValueOf( r.Volume )
// the following is true; val.Kind() != reflect.Prt && val.CanAddr() 
```
Can't currently use `Volume *opt.RepeatedStringOpt` see #9 .

Reply:
This issue should be fixed now as of tag `v0.1.0`